src/components/OptionsForm.tsx

```diff
@@ -16,6 +19,14 @@ export interface OptionsFormProps {
 export function OptionsForm({ options }: OptionsFormProps) {
 	const ownerships = new Set(options.ownership);
 
+	const [isSecondaryOptionsVisible, setIsSecondaryOptionsVisible] = useState(
+		!options.username,
+	);
```

Comment:
```suggestion
	const [optionsExpanded, setOptionsExpanded] = useState(!options.username);
```

Reply:
No worries, I figured that's what you meant! I actually refactored this part of the code to no longer use `useState()` as it's no longer needed, and I'm just using a regular old variable and the name `optionsExpanded` works.